packages/react-day-picker/src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { enGB, enUS } from 'date-fns/locale';
+import { customRender, PageObjects } from 'test';
+
+import { Table } from './Table';
+
+const po = new PageObjects(new Date());
+
+describe('It should render weeks at the boundaries of years correctly', () => {
```

Comment:
I'm not sure this is the test where we should test this behavior. I would add a test to `getWeeks` instead.

Reply:
Good idea. That cleaned things up considerably